Deploy/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+services:
+  db:
+    image: postgres
+    restart: always
+    # set shared memory limit when using docker-compose
+    shm_size: 128mb
+    ports:
+      - '5432:5432'
+    environment:
+      POSTGRES_USER: 'postgres'
+      POSTGRES_PASSWORD: '**your-postgres-pass-here**'
+      POSTGRES_DB: 'b2b-opportunities'
+    volumes:
+      - ./db:/var/lib/postgresql/data
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 10s
+      retries: 5
+      start_period: 30s
+      timeout: 5s
+  app:
+    build:
+      context: ..
+      dockerfile: Deploy/Dockerfile
+    ports:
+      - "8082:8082"
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      POSTGRES_HOST: db
+      DB_USERNAME: 'postgres'
+      DB_PASSWORD: '**your-postgres-pass-here**'
+      JWT_SECRET_KEY: '5nPOeFz6M/0dD6z4b1K9A5XgT5JH5gqF/wW3HrPz6Mg='
```

Comment:
We have to remove this JWT_SECRET_KEY value and use environment (https://docs.docker.com/compose/environment-variables/set-environment-variables/) OR secrets (https://docs.docker.com/compose/use-secrets/) instead